S/SuiteSparse@5/build_tarballs.jl

```diff
@@ -78,9 +60,9 @@ if [[ ${target} == *-apple-* ]] || [[ ${target} == *freebsd* ]]; then
     done
 fi
 
-# Compile suitesparse_wrapper shim
+# Compile SuiteSparse_wrapper shim
 cd $WORKSPACE/srcdir/SuiteSparse_wrapper
-make "${FLAGS[@]}" install
+"${CC}" "${CFLAGS}" -O2 -shared -fPIC -I${prefix}/include SuiteSparse_wrapper.c -o ${libdir}/libsuitesparse_wrapper.${dlext} "{LDFLAGS}" -L${libdir} -lcholmod -lumfpack -lspqr
```

Comment:
Good catch!

Reply:
I think it worked fine, but the handcrafted Makefile was best avoided.